ipfx/data_set_features.py

```diff
@@ -102,6 +114,7 @@ def extractors_for_sweeps(sweep_set,
                           dv_cutoff=20., thresh_frac=0.05,
                           reject_at_stim_start_interval=0,
                           min_peak=-30,
+                          filter_frequency=10.,
```

Comment:
Why was this line added? Does this change the behavior for previously passing processing?